bin/ctags.rb

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env ruby
```

Comment:
I'm not certain your bin/ctags.rb  should go in here - I think you're pretty unique in DLSS and Hydra community in caring about ctags ...